### PR TITLE
feat: modified fit() function in cnn

### PR DESCRIPTION
**Pull request by sameerashahh**
Modified the fit function to evaluate validation losses, and return both the training losses and validation losses.

### CODE REVIEW

File: src/dynadojo/baselines/cnn.py [hunk 1]
```diff
@@ -41,10 +42,15 @@ def fit(self, x: np.ndarray, epochs=100, **kwargs):
                 next_state = self.lin(self.c1(state_t))  # doesn't even call forward
 
                 loss += lossMSE((next_state * (state[:, t, :].unsqueeze(1))), state[:, t+1, :].unsqueeze(1)) #loss_BCEL(next_state, state[:, t + 1, :].unsqueeze(1))
-
+            
             # print(loss.item())
             loss.backward()
             opt.step()
+            losses.append(loss)
```
Comment (carynbear) [inline src/dynadojo/baselines/cnn.py:49]: Should append loss.item()

**Review by carynbear — CHANGES_REQUESTED** [review]

**Review by carynbear — APPROVED** [review]
